data/vibe/data/json.d

```diff
@@ -198,6 +199,8 @@ struct Json {
 	/// ditto
 	this(string v) @trusted { m_type = Type.string; m_string = v; }
 	/// ditto
+	this(UUID v) { this(v.toString()); }
```

Comment:
The support should be restricted to `JsonSerializer` and `JsonStringSerializer`, since the current types are all unambiguously representable as JSON, while `UUID` is really not a JSON value type.